recipes-extended/xen/files/v4v.patch

```diff
@@ -2651,35 +2638,42 @@ PATCHES
 + */
 --- a/xen/include/public/xen.h
 +++ b/xen/include/public/xen.h
-@@ -118,7 +118,7 @@ DEFINE_XEN_GUEST_HANDLE(xen_ulong_t);
+@@ -118,10 +118,11 @@ DEFINE_XEN_GUEST_HANDLE(xen_ulong_t);
  #define __HYPERVISOR_domctl               36
  #define __HYPERVISOR_kexec_op             37
  #define __HYPERVISOR_tmem_op              38
--#define __HYPERVISOR_xc_reserved_op       39 /* reserved for XenClient */
+-#define __HYPERVISOR_argo_op              39
 +#define __HYPERVISOR_v4v_op               39 /* reserved for XenClient */
  #define __HYPERVISOR_xenpmu_op            40
  #define __HYPERVISOR_dm_op                41
+ #define __HYPERVISOR_txt_op               42
++#define __HYPERVISOR_argo_op              43
  
-@@ -178,7 +178,7 @@ DEFINE_XEN_GUEST_HANDLE(xen_ulong_t);
+ /* Architecture-specific hypercall definitions. */
+ #define __HYPERVISOR_arch_0               48
+@@ -178,9 +179,10 @@ DEFINE_XEN_GUEST_HANDLE(xen_ulong_t);
  #define VIRQ_CON_RING   8  /* G. (DOM0) Bytes received on console            */
  #define VIRQ_PCPU_STATE 9  /* G. (DOM0) PCPU state changed                   */
- #define VIRQ_MEM_EVENT  10 /* G. (DOM0) A memory event has occured           */
--#define VIRQ_XC_RESERVED 11 /* G. Reserved for XenClient                     */
+ #define VIRQ_MEM_EVENT  10 /* G. (DOM0) A memory event has occurred          */
+-#define VIRQ_ARGO       11 /* G. Argo interdomain message notification       */
 +#define VIRQ_V4V        11 /* G. Reserved for XenClient                     */
  #define VIRQ_ENOMEM     12 /* G. (DOM0) Low on heap memory       */
  #define VIRQ_XENPMU     13 /* V.  PMC interrupt                              */
++#define VIRQ_ARGO       14 /* G. Argo interdomain message notification       */
```

Comment:
Again, I don't think this is the correct way to patch v4v into Xen 4.12: v4v should be issued the VIRQ number 14, and leave Argo with its VIRQ number 11.